CHANGELOG.md

```diff
@@ -12,10 +12,25 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
-## v1.2.0 (WIP)
+## v2.0.0 (WIP)
+
+- Updated wharf-api-client-go 1.2.0 -> 1.3.0 (#10)
 
 - Added environment var for setting bind address and port. (#14)
 
+- BREAKING: Changed api call order to getToken -> getProvider -> GitHub
+  Operations instead of the previous getProvider -> getToken -> GitHub
+  operations.  (#10)
```

Comment:
Would be nice to have breaking changes first, as those are the most important changes.

Also, could you add a note on that this actually means a different dependency of the Wharf API version?

Such as:

```suggestion
- BREAKING: Changed Wharf API dependency to v4.1.0. This provider now uses call
  order to getToken -> getProvider -> GitHub Operations instead of the previous
  getProvider -> getToken -> GitHub operations.  (#10)
```

Reply:
Taken. And i also reordered the entries such that #10 is a group, version changes together, and breaking changes first.